### ISSUE TITLE
Bug Report: Functions stopped working in the middle of making a song

### ISSUE THREAD

**Issue by 2Schoolhouse** [FNF-CNE-Devs/CodenameEngine#247]
[weed]: <> (MAKE SURE THAT THE ISSUE IS AS COMPLETE AS POSSIBLE. ISSUES THAT HAVE BARELY ANY INFORMATION OR IS ONLY MADE FOR TROLLING WILL BE INSTANTLY CLOSED.)
[weed]: <> (MAKE SURE TO ADD YOUR TARGET OR IT'LL BE IGNORED!)

### Which target are you using?
[weed]: <> (CHANGE [ ] TO [x] IN FRONT OF YOUR TARGET!)

[x ] Windows
[ ] Linux
[ ] Mac

### Explain your issue
[weed]: <> (MAKE SURE TO ADD A SCREENSHOT OF THE CRASH HANDLER, IF AVAILABLE, AND STEPS TO REPRODUCE THE BUG!)

I was making a song playable in Codename (more specifically, a chart of a remix) and I decided to add some little mechanics, stuff like taking away bits of health when the opponent hits a note as an example. However, after finishing up a bit of a portion of the whole chart, some of the functions just... stopped running. Not even reverting to an updated Codename version or reverting the script itself to an older version seems to help. I'm not even sure if it's the chart itself. This affects just the functions, for example onDadHit; the other parts of the script work just fine. This affects both the chart editor and regular play as well.

Is there something I accidentally changed, or was it something else?

**Comment by NexIsDumb** [comment]
hm could you send here the script(s) and a video maybe?

**Comment by 2Schoolhouse** [comment]
Sure. The script itself should go in the/any song's scripts folder if it's gonna be tested, as that is how I made it to be:

https://files.catbox.moe/nkkvcx.hx - The script. Github doesn't seem to want to use the code blocks feature properly so I'm sending it through a catbox link. The functions worked before I got a bit into the chart.

The song itself is a remix of All Stars, titled ENDGAME, by Noichi.

https://streamable.com/h076xc
